common/src/main/java/org/apache/sedona/common/Functions.java

```diff
@@ -447,6 +448,12 @@ public static Geometry lineFromMultiPoint(Geometry geometry) {
         return GEOMETRY_FACTORY.createLineString(coordinates.toArray(new Coordinate[0]));
     }
 
+    public static Geometry closestPoint(Geometry left, Geometry right) {
+        DistanceOp distanceOp = new DistanceOp(left, right);
+        Coordinate[] closestPoints = distanceOp.nearestPoints();
```

Comment:
Since we're using JTS DistanceOp, what happens if the one or both the input geometries are 3D? Is the z ordinate ignored, or it is supported?
We will have to update documentation and tests accordingly.